setup.py

```diff
@@ -49,6 +50,10 @@
         "typing-extensions",
         "Jinja2",
         "pyserial",
+        # Mbed OS 6.6 dependencies
+        "prettytable==0.7.2",
```

Comment:
Can we get away without pinning the versions? I can see dependency hell in our future.

Reply:
We'll have some dependency hell until we have an Mbed Installer, as adding these dependencies now means that the mbed-os version and the mbed-tools version are coupled: you can no longer build any version of Mbed OS 6.5 and up with any mbed-tools version.

Reply:
I've removed the versions from here. Hopefully mbed-os still works.

Reply:
I think we need some more explanation about why we're adding these dependencies in the comment. Also possibly a "TODO: remove this hack when we have an Mbed installer"

Reply:
Commit message expanded with "what if we didn't fix this" description and a link to an issue tracking removal of this usability hack. TODO also added to setup.py itself.